common/util.go

```diff
@@ -605,3 +605,82 @@ func NewCliHelper() CliHelper {
 		cZones:            make(map[string]string),
 	}
 }
+
+var overrideValueByPrefixLogMapLock sync.Mutex
```

Comment:
This API is expected to be called for every GetInputHosts() call, disagree that this is low contention. You are right that you have to check the map twice (on a miss), but that's ok.